yolox_object_detection_node:
  ros__parameters:
    is_debug_mode: true
